docs/machine-learning/how-to-guides/explain-machine-learning-model-permutation-feature-importance-ml-net.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "How-To: Explain model predictions using Permutation Feature Importance"
+description: Understand the feature importance of models with Permutation Feature Importance in ML.NET
+ms.date: 04/30/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc,how-to
+#Customer intent: As a developer, I want to use Permutation Feature Importance in ML.NET to determine the feature importance of my models so that I can understand how my machine learning models make decisions and which features contribute to their performance.
+---
+
+# How-To: Explain model predictions using Permutation Feature Importance
+
+Learn to explain ML.NET machine learning model predictions by understanding the contribution features have to predictions using Permutation Feature Importance (PFI).
+
+Machine learning models are often thought of as black boxes that take inputs and generate an output. The intermediate steps or interactions among the features that influence the output are rarely understood. As machine learning is introduced into more aspects of everyday life such as healthcare, it's of utmost importance to understand why a machine learning model makes the decisions it does. 
```

Comment:
> understand why a machine learning model makes the decisions it does.  [](start = 325, length = 70)

Rich Caruana had a good example in his GAM talk, that might be worth borrowing, to illustrate why it is important.

It is in the Motication section, second column of text here:
[http://people.dbmi.columbia.edu/noemie/papers/15kdd.pdf](http://people.dbmi.columbia.edu/noemie/papers/15kdd.pdf)

Reply:
#resolved